contracts/LBP/LBPWrapper.sol

```diff
@@ -0,0 +1,119 @@
+/*
+██████╗░██████╗░██╗███╗░░░███╗███████╗██████╗░░█████╗░░█████╗░
+██╔══██╗██╔══██╗██║████╗░████║██╔════╝██╔══██╗██╔══██╗██╔══██╗
+██████╔╝██████╔╝██║██╔████╔██║█████╗░░██║░░██║███████║██║░░██║
+██╔═══╝░██╔══██╗██║██║╚██╔╝██║██╔══╝░░██║░░██║██╔══██║██║░░██║
+██║░░░░░██║░░██║██║██║░╚═╝░██║███████╗██████╔╝██║░░██║╚█████╔╝
+╚═╝░░░░░╚═╝░░╚═╝╚═╝╚═╝░░░░░╚═╝╚══════╝╚═════╝░╚═╝░░╚═╝░╚════╝░
+*/
+
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+// solium-disable linebreak-style
+pragma solidity ^0.8.4;
+
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../utils/interface/ILBPFactory.sol";
+import "../utils/interface/IVault.sol";
+import "../utils/interface/ILBP.sol";
+import "hardhat/console.sol";
+
+
+contract LBPWrapper {
+
+    address public owner;
+    address public LBPFactory;
+    address public lbp;
+    uint256 public swapFeePercentage;
+    bool isInitialized;
+
+    modifier onlyOwner{
+        require(msg.sender == owner, "LBPWrapper: only owner function");
+        _;
+    }
+
+    function transferOwnership(address _newOwner) public onlyOwner{
+        require(_newOwner != address(0), "LBPWrapper: new owner cannot be zero");
+        owner = _newOwner;
+    }
+
+    function initialize (
+            address _LBPFactory,
+            uint256 _swapFeePercentage
+    ) public
+    {
+        owner = msg.sender;
+        LBPFactory = _LBPFactory;
+        swapFeePercentage = _swapFeePercentage;
+        isInitialized = true;
+    }
+
+    function deployLbpFromFactory(
+            string memory _name,
+            string memory _symbol,
+            IERC20[] memory _tokens,
```

Comment:
Tokens need to be sorted in numerical order. Is this something we do in the frontend? do we need a check for something like that?

Reply:
We will add a check